ios/example/Hotwire/HotwireCentral.swift

```diff
@@ -11,6 +11,7 @@ class HotwireCentral {
     private init() {
         configureHotwire()
         self.navigator = Navigator()
+        self.navigator.webkitUIDelegate = CustomWKUIController()
```

Comment:
Since we should subclass Hotwire Native's WKUIController, we need to forward the navigator instance as a `WKUIControllerDelegate` to `CustomWKUIController`.

```diff
- self.navigator.webkitUIDelegate = CustomWKUIController()
+ self.navigator.webkitUIDelegate = CustomWKUIController(delegate: navigator)
```